Solutions/Marain.UserNotifications.Management.Host/Marain.UserNotifications.Management.Host.csproj

```diff
@@ -18,13 +18,13 @@
   <ItemGroup>
     <PackageReference Include="Corvus.DotLiquidAsync" Version="1.7.4" />
     <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0" />
-    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.12" />
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.13" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.4.3" />
     <PackageReference Include="Endjin.RecommendedPractices" Version="1.2.0">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="16.9.60">
+    <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="16.10.56">
```

Comment:
Ooh. Fancy! I didn't know we were using this. We should probably get this into the recommended practices component.

Reply:
I've added an issue: https://github.com/endjin/Endjin.RecommendedPractices.AzureDevopsPipelines.GitHub/issues/54